helpers.js

```diff
@@ -11,7 +11,7 @@ const multiply = (number1, number2) => {
 }
 
 const substract = (number1, number2) => {
-  return 1;
+  return number1 - number2;
```

Comment:
```suggestion
  return number2 - number1;
```
This function expects the second minus the first, I think it should be like this, let me know if you can discuss.

Reply:
Whatcha talking bout Willis? this is subtraction, not addition. The ticket request to add number 1 + number 2